sklearn/tests/test_pipeline.py

```diff
@@ -1814,13 +1821,64 @@ def test_pipeline_inverse_transform_Xt_deprecation():
         pipe.inverse_transform(Xt=X)
 
 
+# TODO(1.8): remove this test
```

Comment:
We would need to remove the test but check that we raise an error instead of a warning.